docs/src/docs/components/cldvideoplayer/examples.md

```diff
@@ -0,0 +1,140 @@
+---
+title: CldVideoPlayer/Examples
+order: 3
+---
+
+<script>
+    import Callout from '$lib/components/Callout.svelte'
+    import ImageGrid from '$lib/components/ImageGrid.svelte'
+    import Video from '$lib/components/Video.svelte'
+    import { CldVideoPlayer } from 'svelte-cloudinary'
+
+</script>
+
+# CldVideoPlayer Examples
+
+## Effects
+
+<ImageGrid columns={1}>
+<li>
+  <CldVideoPlayer
+    width="1620"
+    height="1080"
+    src={`videos/mountain-stars`}
+  />
+
+  ### Default
+
+  Basic settings to display a video.
+
+  ```jsx
+  width="1620"
+  height="1080"
+  src="videos/mountain-stars"
+  ```
+</li>
+<li>
+  <div style={{ maxWidth: 500, margin: '0 auto' }}>
+    <CldVideoPlayer
+      width="500"
+      height="500"
+      src={`videos/dog-running-snow`}
+      transformation={{
+        width: 500,
+        height: 500,
+        crop: 'fill',
+        gravity: 'auto'
+      }}
+    />
+  </div>
+
+  ### Crop & Resize
+
+  Basic settings to display a video.
+
+  ```jsx
+  width="1620"
+  height="1080"
+  src="videos/dog-running-snow"
+  ```
+</li>
+<li>
+  <CldVideoPlayer
+    width="1620"
+    height="1080"
+    src={`videos/mountain-stars`}
+    transformation={{
+      overlay: `videos:space-jelly-cosmo-helmet`,
```

Comment:
this is erroring because you have the wrong location. this should be assets:... like the code example right below it

<img width="515" alt="image" src="https://github.com/cloudinary-community/svelte-cloudinary/assets/1045274/c2a99db0-9bc1-4786-a6aa-b4dbff9d0bdc">